Makefile

```diff
@@ -86,8 +86,9 @@ tt:
 	$(CONTROLLER_GEN) object paths="./pkg/controller/resourcegraphdefinition"
 
 .PHONY: fmt
-fmt: ## Run go fmt against code.
+fmt: nwa ## Run go fmt against code and add licenses.
 	go fmt ./...
+	$(NWA) config -c check || $(NWA) config -c add
```

Comment:
The PR which added nwa also configured it to be run from go generate (see codegen.go in the repo root) so if we want the makefile to do it too then the correct way is by calling go generate here.

Reply:
i did hit a go generate lately and it wasn't prepending licenses. I think we should keep it working w/ `go generate` since attribution generation works similarly

Reply:
using `add` seems to work for go generate. Updated the PR. PTAL.